### PR TITLE
Initialize maintenance db

### PR DESCRIPTION
**Pull request by vaskalan**

### CODE REVIEW

File: job_service/adapter/maintenance_db.py [hunk 1]
```diff
@@ -27,33 +26,39 @@ def set_status(status_request: MaintenanceStatusRequest):
     try:
         document = {
             "msg": status_request.msg,
-            "pause": status_request.pause,
+            "paused": status_request.paused,
             "timestamp": str(datetime.now()),
         }
         doc_id = maintenance.insert_one(document)
+        return {"_id": doc_id, **document}
```
Comment (DanielElisenberg) [inline job_service/adapter/maintenance_db.py:33]: we actually usually avoid returning the _id field in the other queries. I think that is reasonable. Maybe we should just return document here 👍🏻

File: tests/unit/api/test_maintenance_api.py [hunk 2]
```diff
@@ -61,8 +61,8 @@ def test_set_maintenance_status_with_no_msg(flask_app, mocker: MockFixture):
     assert "validation_error" in response.json
 
 
-def test_set_maintenance_status_with_invalid_pause(
-    flask_app, mocker: MockFixture
+def test_set_maintenance_status_with_invalid_paused(
+        flask_app, mocker: MockFixture
```
Comment (DanielElisenberg) [inline tests/unit/api/test_maintenance_api.py:65]: 8 space indent => 4 space indent 👀

File: job_service/adapter/maintenance_db.py [hunk 3]
```diff
@@ -30,7 +30,7 @@ def set_status(status_request: MaintenanceStatusRequest):
             "timestamp": str(datetime.now()),
         }
         doc_id = maintenance.insert_one(document)
-        return {"_id": doc_id, **document}
+        return {**document}
```
Comment (DanielElisenberg) [inline job_service/adapter/maintenance_db.py:33]: you can just do a `return document` it will do the same 👍🏻

Comment (vaskalan) [reply job_service/adapter/maintenance_db.py]: Late on a friday evening ... OMG

**Review by DanielElisenberg — APPROVED** [review]
:100:

**Pull request merged**